### PR TITLE
[ETHOSN] Removed support for 20.08 version of the driver stack.

### PR DESCRIPTION
**Pull request by tristan-arm**
- Replaced capabilities header file with api calls introduced by the 20.11 ethosn driver stack release.
  - Removed 20.08 driver stack support and updated all affected code.

### CODE REVIEW

**Comment by tristan-arm** [comment]
Test failure is due to the ci-cpu docker image using an outdated version of the ethos-n driver stack - should work fine if the docker image is recreated to account for changes to [ubuntu_install_ethosn_driver_stack.sh](https://github.com/apache/tvm/blob/main/docker/install/ubuntu_install_ethosn_driver_stack.sh)

@areusch Can look into updating the ci-cpu docker image - Thanks.

**Comment by areusch** [comment]
cc @jroesch -- I think Jared is updating the images, but there is some challenge with GPU drivers right now that is blocking us.

**Comment by tristan-arm** [comment]
cc @leandron @Leo-arm @u99127

**Comment by u99127** [comment]
@mbaret

File: src/relay/backend/contrib/ethosn/codegen_ethosn.h [hunk 1]
```diff
@@ -240,18 +241,14 @@ struct EthosnCompilerConfigNode : public tvm::AttrsNode<EthosnCompilerConfigNode
...
   String compiler_algorithm;
 
   TVM_DECLARE_ATTRS(EthosnCompilerConfigNode, "ext.attrs.EthosnCompilerConfigNode") {
-    TVM_ATTR_FIELD(variant)
-        .describe(
-            "0 for Ethos-N77, 1 for Ethos-N57, 2 for Ethos-N37,"
-            " 3 for Ethos-N78. See Ethos-N documentation.")
+    TVM_ATTR_FIELD(variant).describe("See Ethos-N documentation.").set_default("Ethos-N77");
+    TVM_ATTR_FIELD(sram_size_bytes)
+        .describe("Optionally override the default sram size. See Ethos-N documentation.")
```
Comment (mbaret) [inline src/relay/backend/contrib/ethosn/codegen_ethosn.h:251]: Is defaulting the sram size to 0 the correct thing to do?

Comment (tristan-arm) [reply src/relay/backend/contrib/ethosn/codegen_ethosn.h:251]: Yes, zero is used by the ethos-n driver stack to indicate that the default sram size is not to be overridden.

**Comment by areusch** [comment]
@jroesch what's the status of ci-cpu update?

**Comment by areusch** [comment]
@mbaret @leandron @u99127 @leo-arm the CI container has been updated, please take a look at this PR when you have a chance!

**Review by mbaret — APPROVED** [review]
LGTM, but @tristan-arm could you confirm this has been tested against upstream CI post the container update before I merge?

**Comment by tristan-arm** [comment]
> LGTM, but @tristan-arm could you confirm this has been tested against upstream CI post the container update before I merge?

Sure, I've rebased and the tests are now green against the latest container updates.

**Comment by mbaret** [comment]
This is now merged. Thanks @tristan-arm

**Pull request merged**